provider-management.md

```diff
@@ -0,0 +1,34 @@
+# Provider Management
+
+![Provider Manager Dashboard](assets/provider_manager_dashboard.png)
```

Comment:
Add a brief sentence to show what this module does. E.g. 'The Provider Management module is where you may get or update info on your roles as a provider, and see the roles of other providers in the system.